index.html

```diff
@@ -224,12 +293,12 @@ <h1>About me</h3>
             <div class="form-group">
                 <input class="input" type="text" name="name" maxlength="30" placeholder="Full name" rows="10" required>
                 <input class="input" type="email" name="email" placeholder="Email address" required>
-                <textarea name="send" maxlength="500" required placeholder="Write me something"></textarea>
+                <textarea name="send" maxlength="399" required>Write me something</textarea>
                 <input class="submit" type="submit" value="Get in touch">
             </div>
         </form>
     </section>
-    <hr>
+    <hr class="hr">
     <footer class="footer">
```

Comment:
- [ ] Kindly adjust the text and the form alignment on your design to match the figma design.
- [ ] You also need to add the social media lists that are missing at the bottom of your design.

### Your Design Desktop
![your-form-desktop](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/adc85a53-0003-43e6-a0ed-029c2d0f5308)

### Figma Design Desktop
![temp-form-desktop](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/5b30414b-e2af-4f24-9daf-c2a5b16159d4)

- [ ] You also need to work on the style for the social media on the mobile.

### Your Design
[app-behave-form.webm](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/6eb3967a-06e2-429b-9702-f11b558325eb)


### Figma Design
![temp-form](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/2880feef-e850-42cc-b9c1-e20ff83e3891)